csharp/ql/test/library-tests/controlflow/graph/ExtractorOptions.cs

```diff
@@ -1 +1,2 @@
 // semmle-extractor-options: /langversion:8.0
+#nullable enable
```

Comment:
Why this addition?

Reply:
With this setting, unannotated reftypes are , andIt enables the feature in Roslyn. By default, flow state is not computed, and the types are assumed to be nullable.

Reply:
With this setting, unannotated reftypes are non-nullable, and flow state is computed.  Without this settings, unannotated reftypes are nullable and flow state is not computed. 

We want to enable nullability for this test to make it interesting.

Reply:
Still, this is the only change to `library-tests/controlflow/graph`, I thought we only needed it for `library-tests/controlflow/guards` (where it is also added [here](https://github.com/Semmle/ql/pull/2450/files?file-filters%5B%5D=.cs#diff-3d93762a28c88302b14baf20534d06dfR2)).

Reply:
It might be spurious -- let me check.